cmd/install/assets/hypershift_operator.go

```diff
@@ -913,6 +913,14 @@ func (o HyperShiftOperatorClusterRole) Build() *rbacv1.ClusterRole {
 				Resources: []string{rbacv1.ResourceAll},
 				Verbs:     []string{rbacv1.VerbAll},
 			},
+			{
+				APIGroups: []string{
+					"cluster.x-k8s.io",
+				},
+				Resources: []string{"machines/finalizers"},
+				Verbs:     []string{"update"},
+			},
+
```

Comment:
can we remove this?

Reply:
If we want to ahve `blockOwnerDelete` we need this 

https://sdk.operatorframework.io/docs/faqs/#after-deploying-my-operator-why-do-i-see-errors-like-is-forbidden-cannot-set-blockownerdeletion-if-an-ownerreference-refers-to-a-resource-you-cant-set-finalizers-on-